src/Widgets/HeaderBar.vala

```diff
@@ -129,19 +155,26 @@ public class Camera.Widgets.HeaderBar : Gtk.HeaderBar {
         var image_settings = new Gtk.Grid () {
             column_spacing = 6,
             row_spacing = 3,
-            margin = 12,
-            width_request = 250
+            margin = 12
         };
-        image_settings.attach (brightness_image, 0, 0);
-        image_settings.attach (brightness_label, 1, 0);
-        image_settings.attach (brightness_scale, 0, 1, 2);
-        image_settings.attach (contrast_image, 0, 2);
-        image_settings.attach (constrast_label, 1, 2);
-        image_settings.attach (contrast_scale, 0, 3, 2);
+        image_settings.attach (brightness_image, 0, 1);
+        image_settings.attach (brightness_label, 1, 1);
+        image_settings.attach (brightness_scale, 0, 2, 3);
+        image_settings.attach (contrast_image, 0, 3);
+        image_settings.attach (constrast_label, 1, 3);
+        image_settings.attach (contrast_scale, 0, 4, 3);
```

Comment:
We probably don't need to change these since we don't pack anything else into this grid :)